packages/@expo/cli/src/start/doctor/dependencies/validateDependenciesVersions.ts

```diff
@@ -252,3 +253,8 @@ function findDependencyType(
 
   return 'dependencies';
 }
+
+/** Check if the currently installed `expo` version is a pre-released (canary) version */
+async function isExpoPreReleaseAsync(projectRoot: string) {
+  return semver.prerelease(await getPackageVersionAsync(projectRoot, 'expo')) !== null;
```

Comment:
Tbh I always feel safer with `==` instead of `===` when comparing to `null` because it also handles `undefined` pretty well.

```suggestion
  return semver.prerelease(await getPackageVersionAsync(projectRoot, 'expo')) != null;
```

Reply:
How about this instead?

```suggestion
  return !!semver.prerelease(await getPackageVersionAsync(projectRoot, 'expo'));
```

Reply:
Yeah, it's fine as well 👍